src/main/offline/io/stargate/sgv2/jsonapi/service/cqldriver/sstablewriter/OfflineFileWriterInitializer.java

```diff
@@ -0,0 +1,27 @@
+package io.stargate.sgv2.jsonapi.service.cqldriver.sstablewriter;
+
+import static io.stargate.sgv2.jsonapi.service.cqldriver.CQLSessionCache.OFFLINE_WRITER;
+
+import io.smallrye.config.SmallRyeConfig;
+import io.smallrye.config.SmallRyeConfigBuilder;
+import io.smallrye.context.SmallRyeContextManagerProvider;
+import io.stargate.sgv2.jsonapi.config.OperationsConfig;
+import io.stargate.sgv2.jsonapi.config.constants.ApiConstants;
+
+public class OfflineFileWriterInitializer {
+
+  public static void initialize() {
+    System.setProperty(ApiConstants.OFFLINE_WRITER_MODE_PROPERTY, "true");
```

Comment:
This seems.... somehow wrong. System properties should not usually be changed by running Java code. Especially when there's timing question (something having to call initialize()).
Is there no other way to achieve this?

Reply:
I agree with you on the timing part. Since, invoking this `initialize` method one time during start up is a 'must' for the offline sstable writer, I thought of leaving it here. Apart from this vm arg, invoking `instance()` method of `SmallRyeContextManagerProvider` is also required during start up and so I put them together in this startup method.